photon-client/src/components/dashboard/tabs/TargetsTab.vue

```diff
@@ -78,6 +76,23 @@ import { useStateStore } from "@/stores/StateStore";
         </template>
       </v-simple-table>
     </v-row>
+    <v-row align="start" class="pb-4 white--text">
```

Comment:
This entire section should be hidden if the `doMultiTarget` prop is false or undefined as it will always be empty in non-tag pipelines or when multitag is disabled